pyvista/utilities/__init__.py

```diff
@@ -15,6 +15,13 @@
 from .fileio import *
 from .geometric_objects import *
 from .helpers import *
+from .misc import (
+    PyVistaDeprecationWarning,
+    _get_vtk_id_type,
+    _set_plot_theme_from_env,
+    set_pickle_format,
+    vtk_version_info,
+)
```

Comment:
I've added this block because it was missing. This makes the API surface here _larger_, but it seemed more consistent with the _status quo_, and we can still remove what's unnecessary here later in the life of this PR. This also shows how names can be redundant: since we're using `from pyvista.utilities.misc import ...` in `pyvista.__init__`, we don't really have to have these here.

Depending on how we decide to structure the higher-level inits, we might end up with very lean subpackage inits.